loginModuleJava/src/main/java/com/brandon/loginmodulejava/viewmodel/LegacyLoginViewModel.java

```diff
@@ -0,0 +1,57 @@
+package com.brandon.loginmodulejava.viewmodel;
+
+
+import static com.brandon.corejava.utilities.ConnectionErrors.NO_CONNECTION_ERROR;
+import static com.brandon.corejava.utilities.UserInputErrors.NO_INPUT_ERROR;
+import static com.brandon.loginmodulejava.viewmodel.LoginUiStates.IDLE;
+
+import android.net.ConnectivityManager;
+import android.util.Log;
+
+import com.brandon.loginmodulejava.domain.LoginUseCaseJava;
+
+import javax.inject.Inject;
+
+import dagger.hilt.android.lifecycle.HiltViewModel;
+import io.reactivex.rxjava3.annotations.NonNull;
+import io.reactivex.rxjava3.core.BackpressureStrategy;
+import io.reactivex.rxjava3.core.Flowable;
+import io.reactivex.rxjava3.subjects.BehaviorSubject;
+
+@HiltViewModel
+public class LegacyLoginViewModel extends LoginViewModel {
+    private final LoginUseCaseJava loginUseCaseJava;
+    private final ConnectivityManager connectivityManager;
+
+    private final BehaviorSubject<LoginUiStateJava> _uiState = BehaviorSubject.createDefault(new LoginUiStateJava("", NO_INPUT_ERROR, NO_CONNECTION_ERROR, false, IDLE));
```

Comment:
Prefix underscores are a kotlin convention for field backing variables. No need to leave the underscore here

Reply:
removed underscore.